src/main/python/core/controller.py

```diff
@@ -624,8 +624,13 @@ def build_oc_tree(self, tree: QTreeWidgetItem) -> None:
     def init_overclock_tree(self):
         self.widget.overclock_tree.clear()
         overclock_tree = self.widget.overclock_tree.invisibleRootItem()
-        self.build_oc_tree(overclock_tree)
-        self.widget.overclock_tree.sortItems(0, Qt.SortOrder.AscendingOrder)
+        if self.state_manager.get_max_promo() > 0:
+            self.build_oc_tree(overclock_tree)
+            self.widget.overclock_tree.sortItems(0, Qt.SortOrder.AscendingOrder)
+        else:
+            error_text = QTreeWidgetItem(None)
+            error_text.setText(0, "No dwarf promoted yet")
+            overclock_tree.addChild(error_text)
```

Comment:
wouldn't
```suggestion
            error_text = QTreeWidgetItem(overclock_tree)
            error_text.setText(0, "No dwarf promoted yet")
```
have the same effect?

Reply:
Indeed, fixing right now